packages/back-end/src/services/features.ts

```diff
@@ -184,3 +189,135 @@ export function verifyDraftsAreEqual(
     );
   }
 }
+
+function parseDefaultValue(
+  defaultValue: string,
+  valueType: FeatureValueType
+): string {
+  if (valueType === "boolean") {
+    return defaultValue === "true" ? "true" : "false";
+  }
+  if (valueType === "number") {
+    return parseFloat(defaultValue) + "";
+  }
+  if (valueType === "string") {
+    return defaultValue;
+  }
+  try {
+    return JSON.stringify(JSON.parse(defaultValue), null, 2);
+  } catch (e) {
+    throw new Error(`JSON parse error for default value`);
+  }
+}
+
+export async function createFeatureService(feature: Partial<FeatureInterface>) {
```

Comment:
The name of this function is a little weird.  It sounds like it's creating a service.